html/rendering/non-replaced-elements/the-hr-element-0/setting-overflow-visible.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<title>The hr element: setting 'overflow: visible'</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#test-visible {
+  overflow: visible;
+}
+.float, hr::before {
+  content: "";
+  float: left;
+  width: 50px;
+  height: 50px;
+  background-color: orange;
+}
+.clear {
+  clear: left;
+}
+</style>
+
+<div class=float></div>
+<hr id=test-control>
+<div class=clear></div>
+
+<div class=float></div>
+<hr id=test-visible>
```

Comment:
Would be useful with an additional floated sibling of the HR, to make sure that the float inside the HR affects that float correctly.